en/developer_guide/install_with_vagrant_for_contributors.rst

```diff
@@ -0,0 +1,223 @@
+Install OpenOrchestra with vagrant for contributors
+===================================================
+
+Open Orchestra is composed of three projects: 
+
+- *open-orchestra* which is the CMS Back Office;
+- *open-orchestra-front-demo* which is the Front Office part that will display the sites and pages
+  created in the CMS Back Office;
+- *open-orchestra-media-demo* which will display the images loaded in the CMS Back Office.
+
+For contributors, we provide a Vagrant-powered environment with provisioning so you get minimal
+setup actions to do.
+
+Install Vagrant
+---------------
+The project is running on a Vagrant virtual environment built on VirtualBox to be production ready.
+
+.. code-block:: bash
+
+    $ aptitude install virtualbox
+    $ wget https://dl.bintray.com/mitchellh/vagrant/vagrant_1.6.3_x86_64.deb
+    $ dpkg -i vagrant_1.6.3_x86_64.deb
+
+Install ansible
+---------------
+
+All the project server configuration is going to be handled by ansible.
+
+Our configuration was made for the 1.9.4 version of ansible.
+
+For more information about ansible installation, you can see `Ansible installation documentation`_.
+
+
+Install nfs server
+------------------
+
+To improve the vagrant box performance, we share the folders with the ``nfs`` protocol. You need to
+install a nfs server instance on your computer.
+
+.. code-block:: bash
+
+    $ aptitude install nfs-kernel-server
+
+Download Composer
+-----------------
+
+Composer is the package manager used by modern PHP applications.
+
+To install composer with curl:
+
+.. code-block:: bash
+
+    $ curl -sS https://getcomposer.org/installer | php
+
+If you don't have curl installed, you can also download it with php:
+
+.. code-block:: bash
+
+    $ php -r "readfile('https://getcomposer.org/installer');" | php
+
+see `Download Composer`_
+
+Install OpenOrchestra
+---------------------
+To contribute on the project you have to install two versions: the master version and the last 
+stable one. Here is the directory tree with your two projects that we recommand:
```

Comment:
Maybe you should mention that the master branch is the dev branch for future features